lib/init/config-initializer.js

```diff
@@ -445,114 +450,121 @@ function askInstallModules(modules, packageJsonExists) {
             name: "moduleType",
             message: "What type of modules does your project use?",
-            default: "esm",
+            initial: 0,
             choices: [
-                { name: "JavaScript modules (import/export)", value: "esm" },
-                { name: "CommonJS (require/exports)", value: "commonjs" },
-                { name: "None of these", value: "none" }
+                { message: "JavaScript modules (import/export)", name: "esm" },
+                { message: "CommonJS (require/exports)", name: "commonjs" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "framework",
             message: "Which framework does your project use?",
-            default: "react",
+            initial: 0,
             choices: [
-                { name: "React", value: "react" },
-                { name: "Vue.js", value: "vue" },
-                { name: "None of these", value: "none" }
+                { message: "React", name: "react" },
+                { message: "Vue.js", name: "vue" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "confirm",
+            type: "toggle",
             name: "typescript",
             message: "Does your project use TypeScript?",
-            default: false
+            enabled: "Yes",
+            disabled: "No",
+            initial: 0
         },
         {
-            type: "checkbox",
+            type: "multiselect",
             name: "env",
             message: "Where does your code run?",
-            default: ["browser"],
+            initial: 0,
             choices: [
-                { name: "Browser", value: "browser" },
-                { name: "Node", value: "node" }
+                { message: "Browser", name: "browser" },
+                { message: "Node", name: "node" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "source",
             message: "How would you like to define a style for your project?",
-            default: "guide",
             choices: [
-                { name: "Use a popular style guide", value: "guide" },
-                { name: "Answer questions about your style", value: "prompt" },
-                { name: "Inspect your JavaScript file(s)", value: "auto" }
+                { message: "Use a popular style guide", name: "guide" },
+                { message: "Answer questions about your style", name: "prompt" },
+                { message: "Inspect your JavaScript file(s)", name: "auto" }
             ],
-            when(answers) {
-                return answers.purpose === "style";
+            skip() {
+                return this.state.answers.purpose !== "style";
+            },
+            result(input) {
+                return this.skip ? null : input;
```

Comment:
@Siilwyn 
It seems not working as we expected.
When I answered with the "guide," in this step.  but the [selecting styleguide step](https://github.com/eslint/eslint/pull/13254/files#diff-bc3fbe576813fc2a16ef2036becd32f8R533) was skipped.

It seems the `this.skip` is a function (`truthy` value), so the result always be a `null`


![스크린샷 2020-05-05 오후 10 19 31](https://user-images.githubusercontent.com/41323220/81070764-aa77f300-8f1e-11ea-9ffa-1ea8a70d8870.png)